ArduPlane/GCS_Mavlink.cpp

```diff
@@ -699,6 +699,8 @@ bool GCS_MAVLINK_Plane::try_send_message(enum ap_message id)
     case MSG_AOA_SSA:
         CHECK_PAYLOAD_SIZE(AOA_SSA);
         plane.send_aoa_ssa(chan);
+    case MSG_LANDING:
+        plane.landing.send_landing_message(chan);
```

Comment:
There are examples of both approaches in the code. For example mount will always get called regardless of if there are no mounts, send_battery2 is always called and lets the backend manage if it's their or not. I honestly think the one that lets the backend library decide if it should be sending or not is the better model as it allows for more flexibility.